keras_hub/src/models/yolo_v8/yolo_v8_backbone.py

```diff
@@ -0,0 +1,193 @@
+import keras
+from keras import ops
+from keras.layers import Input
+from keras.layers import MaxPooling2D
+from keras.layers import Rescaling
+
+from keras_hub.src.api_export import keras_hub_export
+from keras_hub.src.models.backbone import Backbone
+from keras_hub.src.models.yolo_v8.yolo_v8_layers import apply_conv_bn
+from keras_hub.src.models.yolo_v8.yolo_v8_layers import apply_csp_block
+
+
+def get_tensor_input_name(tensor):
+    return tensor._keras_history.operation.name
+
+
+def parse_model_inputs(input_shape, input_tensor, **kwargs):
+    if input_tensor is None:
+        return Input(shape=input_shape, **kwargs)
+    else:
+        if not keras.backend.is_keras_tensor(input_tensor):
+            return Input(tensor=input_tensor, shape=input_shape, **kwargs)
+        else:
+            return input_tensor
+
+
+def apply_spatial_pyramid_pooling_fast(
+    inputs, pool_size=5, activation="swish", name="spp_fast"
+):
+    channel_axis = -1
+    input_channels = inputs.shape[channel_axis]
+    hidden_channels = int(input_channels // 2)
+
+    x = apply_conv_bn(
+        inputs,
+        hidden_channels,
+        kernel_size=1,
+        activation=activation,
+        name=f"{name}_pre",
+    )
+    pool_1 = MaxPooling2D(
+        pool_size=pool_size, strides=1, padding="same", name=f"{name}_pool1"
+    )(x)
+    pool_2 = MaxPooling2D(
+        pool_size=pool_size, strides=1, padding="same", name=f"{name}_pool2"
+    )(pool_1)
+    pool_3 = MaxPooling2D(
+        pool_size=pool_size, strides=1, padding="same", name=f"{name}_pool3"
+    )(pool_2)
+
+    out = ops.concatenate([x, pool_1, pool_2, pool_3], axis=channel_axis)
+    out = apply_conv_bn(
+        out,
+        input_channels,
+        kernel_size=1,
+        activation=activation,
+        name=f"{name}_output",
+    )
+    return out
+
+
+@keras_hub_export("keras_hub.models.YOLOV8Backbone")
+class YOLOV8Backbone(Backbone):
+    """Implements the YOLOV8 backbone for object detection.
+
+    This backbone is a variant of the `CSPDarkNetBackbone` architecture.
+
+    For transfer learning use cases, make sure to read the
+    [guide to transfer learning & fine-tuning](https://keras.io/guides/transfer_learning/).
+
+    Args:
+        stackwise_channels: A list of ints, the number of channels for each dark
```

Comment:
NIT : the format to add arg type is 
stackwise_channels: A list of int.
include_rescaling: bool. Rescale inputs is set to True...
activation: str. The activation function....